base/build.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
```

Comment:
Could you convert it Gradle Kotlin DSL. `build.gradle.kts`
You can get reference from other gradle.kts files.